spocc_download.R

```diff
@@ -0,0 +1,54 @@
+################################################################################
+################################################################################
+##################### Downloading Occurrences via SPOCC ########################
+################################################################################
+################################################################################
+
+# libraries
+library(spocc)
+library(sf)
+library(tidyverse)
```

Comment:
This is more of a overall pipeline question - do we want to load all our libraries in setup.R and then just run that at the top of every script?